flow/size.yaml

```diff
@@ -62,3 +67,70 @@ components:
           type: integer
           default: 1518
           x-field-uid: 2
+    Flow.SizeWeightPairs:
+      type: object
+      description: |-
+        Frame size distribution, defined as <size, weight> pairs (including IMIX distribution).
+        Frames are randomly generated such that the proportion of each frame size out of the total number of frames 
+        are matching with the weight value of the <size, weight> pair. However, as with any other probability 
+        distribution, the sample distribution is close to theoretical value only if the size of the sample is reasonably large. 
+        When the number of frames is very low the transmitted frames may not come close to the ratio described in the weight.
+      properties:
+        choice:
+          type: string
+          default: predefined
+          x-field-uid: 1
+          x-enum:
+            predefined:
+              x-field-uid: 1
+            custom:
+              x-field-uid: 2
+        predefined:
+          type: string
+          default: imix
+          description: |-
+            Specify predefined frame size distribution <size, weight> pairs (including IMIX distribution).  
+            The available predefined distribution pairs are:
+            - IMIX (64:7, 570:4, and 1518:1)  
```

Comment:
The weights of the IMIX distribution do not add up to 100, while the weights in all the other pre-defined distributions add up to 100.

Reply:
Predefined distributions are specified in %, but custom weight pairs are more flexible, and any distribution can be easily achieved. For example, if we have a sample size of 4M packets, and we know the packet count for each distinct frame size, the same can be easily translated to the custom pattern, where the sum of weights will be 4M.

Reply:
Ok.  No issues with not being % and not adding up to 100.  That is fine, just one minor nit - IMIX is pre-defined and not specified like the other pre-defined distributions.

Reply:
All distributions are pre-defined, under the "predefined" category.  The user needs to select one of the available choices. So,  should we remove the word "Specify" from the description to read it like "Predefined frame size distribution ..." or add one more line in the description "IMIX is pre-defined and not specified like the other pre-defined distributions."? Could you please help with the exact change/wording on the nit?